org_chart.js

```diff
@@ -4,70 +4,42 @@ let elon = {
   position: "Boss",
 }
 
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
-  }
-]
 
-// assign each a manager
-let [alan, grace, donald] = new_hires
+function Employee(name, email, position, project, manager){
+  this.name = name;
+  this.email = email;
+  this.position = position;
+  this.project = project;
+  this.manager = manager;
+}
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+let alan = new Employee("Alan Turing", "grace.hopper@musky.com", "Computer Scientist", "Cryptography", elon)
+let grace = new Employee("Grace Hopper", "alan.turing@musky.com", "Computer Scientist", "Compilers", elon)
+let donald = new Employee("Donald Knuth", "donald.knuth@musky.com", "Mathematician", "Algorithm Analysis", elon)
 
-// send to training
-if (!alan.skills) {
-  alan.skills = []
-}
+
+alan.skills = []
 alan.skills.push("Mathematics")
 
-if (!grace.skills) {
-  grace.skills = []
-}
+grace.skills = []
 grace.skills.push("Cobol")
 
-if (!donald.skills) {
-  donald.skills = []
-}
+donald.skills = []
```

Comment:
Prototype functions should handle this.  Then you can remove even more repetition